src/components/FieldErrors/WithFieldErrorsHOC.js

```diff
@@ -10,140 +10,136 @@ import {
 // it provides a source of truth for displaying errors and disabling/enabling action buttons
 // it passes down the component tree 'fieldErrorsWrapper' object with HOC-specific props along with parent props
 //
-const withFieldErrors = WrappedComponent => {
-  return class WithFieldErrors extends PureComponent {
-    static propTypes = {
-      model: PropTypes.shape({
-        data: PropTypes.shape({
-          fieldErrors: PropTypes.object,
-          viewName: PropTypes.string
-        }),
-        actions: PropTypes.objectOf(PropTypes.func)
-      }).isRequired
-    }
+export default WrappedComponent => class WithFieldErrors extends PureComponent {
```

Comment:
Full redesign is required for wrapping particular Edits only.

Overwritten props:
* errors

Swallowed props:
* suppressErrorTriggers
* display/looseErrorTriggers

Reply:
Subject for a separate issue.